### PR TITLE
Add mutate.R and setLabels.R

### PR DESCRIPTION
**Pull request by eoduniyi**
Please see issue #12 for knowing what mutate.R allows and disallows

### CODE REVIEW

**Comment by cbeleites** [comment]
@eoduniyi the build failed because there's a leftover old package name in your branch. 

I already found & fixed it in develop, so I merged develop into your feature branch and resolved the conflicts

File: R/setLabels.R [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+setLabels.transmute <- function(.data, ...) {
+    setLabels(.data, ...)
+}
+
+#' @rdname setLabels
+#' @aliases setLabels
+#' @param .data hyperSpec object
+#' @param res resulting data frame
+#' @export
+setLabels.select <- function(.data, res) {
```
Comment (cbeleites) [inline R/setLabels.R]: - Should `setLabels.select()` and `setLabels.rename()` really be exported?
- since `method.class()` signifies S3 methods, would snake-case names (`setLabels_select()`) be better here?

File: R/transmute.R [hunk 2]
```diff
@@ -1,138 +1,56 @@
...
+.test(transmute.hyperSpec) <- function() {
   context("transmute.hyperSpec")
 
-  # UT1
   test_that("non hyperSpec objects are rejected", {
     df <- data.frame(a = NA, b = NA)
     expect_error(transmute.hyperSpec(df))
   })
 
-  # UT2
```
Comment (cbeleites) [inline R/transmute.R:102]: @eoduniyi, why did you remove the unit tests?

Comment (eoduniyi) [reply R/transmute.R:102]: UT2 was:
```R
  # UT2
  test_that("arguments are correctly passed onto mutate/transmute", {
    df <- flu
    df <- dplyr::transmute(df@data, c, c = 0, c = 1)
    expect_equivalent(transmute.hyperSpec(flu, c, c = 0, c = 1), df)
  })
```

But making sure arguments are passed in their correct order is behaviour that is the responsibility of `dplyr`

Comment (cbeleites) [reply R/transmute.R:102]: Sure, but we still want to check that we get the correct result. We *could* have a mistake that would return an invalid hyperSpec object, or the wrong content e.g. if our code doesn't hand over the correct result form dplyr.

File: R/setLabels.R (hunk 1 above)
Comment (eoduniyi) [reply R/setLabels.R]: Okay, will change to snake-case
